src/Components/Calculator.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from "react";
+import "./Calculator.css";
+
+class Calculator extends Component {
+  state = {
+    output: "0",
+    lastEntry: null,
+    waitingForEntry: false,
+    operator: null,
+  };
+
+  inputNum = (num) => {
+    const { output, waitingForEntry } = this.state;
+    let number = String(num);
```

Comment:
Turning this into a string won't be necessary if you instead e.target.value because each of your buttons has a value that is equal to a string of the value